addons/website/static/tests/tours/website_seo_notification.js

```diff
@@ -74,6 +75,13 @@ registerWebsitePreviewTour(
             trigger: ":iframe #o_main_nav .js_usermenu a.dropdown-item.ps-3:contains('My Account')",
             run: "click",
```

Comment:
```suggestion
            import { waitFor } from "@odoo/hoot-dom";

            async run(actions) {
                await actions.click();
                await waitFor(":iframe #wrapwrap", { timeout: 8000 });
            },
```

Can you remove following extra step and try something like this?